src/main/java/com/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,201 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.exceptions.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        String query = "SELECT order_id, user_id FROM orders "
+                + " WHERE user_id = ? AND deleted = FALSE";
+        List<Order> orders = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, userId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Order order = getOrderFromResultSet(resultSet);
+                orders.add(order);
+            }
+            return orders;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Getting user orders by user id "
+                    + userId + " failed. ", e);
+        }
+    }
+
+    @Override
+    public Order create(Order order) {
+        String query = "INSERT INTO orders (user_id) VALUES (?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            Order newOrder = new Order(order.getUserId(), order.getProducts());
+            PreparedStatement statement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
            PreparedStatement statement 
                               = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
```
Everywhere) I was told previously that only streams can be separated this way